src/plugins/plugin.legend.js

```diff
@@ -411,15 +412,30 @@ var Legend = Element.extend({
 			// Horizontal
 			var isHorizontal = me.isHorizontal();
 			if (isHorizontal) {
+				// default to center as this is the original behaviour of just position
+				var alignedX = ((legendWidth - lineWidths[0]) / 2) + labelOpts.padding;
+				if (opts.alignment === 'left') {
+					alignedX = 0;
+				} else if (opts.alignment === 'right') {
+					alignedX = ((legendWidth - lineWidths[0])) + labelOpts.padding;
```

Comment:
Similar to my comment on the vertical case, I don't think `legendWidth` is the right edge of the legend box in pixels, instead `me.right` would be correct.

Reply:
I had not thought about the tests but in looking through all the current tests, the most obvious ones would be to change the `should draw correctly when the legend is positioned on the top` test into 3 tests:

1. should draw correctly when the legend is positioned on the top and align start
2. should draw correctly when the legend is positioned on the top and align center (this would be the same test as it currently is)
3. should draw correctly when the legend is positioned on the top and align end

And then something similar for the `should draw correctly when the legend is positioned on the left`, `should draw correctly when the legend is positioned on the top and has multiple rows`, and `should draw correctly when the legend is positioned on the left and has multiple columns` tests